pkg/projects/resource_project.go

```diff
@@ -215,9 +215,16 @@ func projectResource() *schema.Resource {
 			},
 			Description: "Project role. Element has one to one mapping with the [JFrog Project Roles API](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-AddaNewRole)",
 		},
+
+		"repos": {
+			Type:        schema.TypeSet,
+			Optional:    true,
+			Elem:        &schema.Schema{Type: schema.TypeString},
```

Comment:
I wonder: You declare the type as string in `Elem` can you put a validator in here?

Reply:
Tried it, didn't work. Will compile but panic at runtime.